tests/benchmark/data/hnsw_indices/hnsw_indices_all.txt

```diff
@@ -1,14 +1,14 @@
-https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/DBpedia-n1M-cosine-d768-M64-EFC512.hnsw_v1
-https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/DBpedia-test_vectors-n10k.raw
+https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/dbpedia-cosine-dim768-M64-efc512.hnsw_v3
```

Comment:
a special reason for omitting the dataset size?

Reply:
No. I tried to make a standard naming template. We can add the data size to all the files, I tried to keep the most relevant parameters only (those you need to know in order to create the index - dim, type, metric, and M and efc)